redash/models.py

```diff
@@ -244,6 +244,13 @@ def all(cls, org):
     def members(cls, group_id):
         return User.select().where(peewee.SQL("%s = ANY(groups)", group_id))
 
+    @classmethod
+    def find_by_name(cls, org, group_names):
+        if len(group_names) == 0:
+            return []
+        result = cls.select().where(cls.org == org, cls.name << group_names)
+        return list(result) if result.count() > 0 else []
```

Comment:
`result.count()` will do another query... wouldn't it return an empty list anyway?